pkg/kv/kvserver/replica_raft.go

```diff
@@ -181,6 +181,7 @@ func (r *Replica) evalAndPropose(
 	// granularity of admission control doing token and size estimation (which
 	// is 15s). Also, admission control corrects for gaps in reporting.
 	writeBytes := kvadmission.NewStoreWriteBytes()
+	// TODO(msbutler): anything to do here for external sst linking?
```

Comment:
i believe its fine to tell admission control that the writeBytes and ingestedBytes is 0.

Reply:
yes.